ownership/src/interface.sw

```diff
@@ -0,0 +1,21 @@
+library interface;
+
+/// Intended to be logged when ownership is transferred from
+/// `previous_owner` to `new_owner`.
+pub struct OwnershipTransferredEvent {
+    previous_owner: Option<Identity>,
+    new_owner: Option<Identity>,
+}
+
+/// An Ownable contract.
+abi Ownable {
+    /// Gets the current owner.
+    #[storage(read)]
+    fn owner() -> Option<Identity>;
```

Comment:
why option?

Reply:
I suppose so it can be transferred to None?

Reply:
Yeah. Just going off the suggested best practice here https://fuellabs.github.io/sway/master/blockchain-development/access_control.html#contract-ownership